charts/mysqldump/templates/configmap-script.yaml

```diff
@@ -39,27 +39,32 @@ data:
     {{ end -}}
     {{ end -}}
 
-    {{ if and (.Values.mysql.db) (eq .Values.allDatabases.enabled false) }}
-    MYSQL_DB="{{ .Values.mysql.db }}"
-    echo "Backing up single db ${MYSQL_DB}"
-    {{ if .Values.saveToDirectory }}mkdir -p "${BACKUP_DIR}"/"${MYSQL_DB}"{{ end }}
-    mysqldump ${MYSQL_OPTS} -h ${MYSQL_HOST} -P ${MYSQL_PORT} -u ${MYSQL_USERNAME} --databases ${MYSQL_DB} | gzip > ${BACKUP_DIR}/{{ if .Values.saveToDirectory }}${MYSQL_DB}/{{ end }}${TIMESTAMP}_${MYSQL_DB}.sql.gz
-    rc=$?
-    {{ else if and (.Values.allDatabases.enabled) (eq .Values.allDatabases.singleBackupFile false)}}
-    for MYSQL_DB in $(mysql -h "${MYSQL_HOST}" -P ${MYSQL_PORT} -u ${MYSQL_USERNAME} -B -N -e "SHOW DATABASES;"|egrep -v '^(information|performance)_schema$'); do
+    {{ if (eq .Values.allDatabases.singleBackupFile false) }}
```

Comment:
please use "and" instead of 2 times "if" in a row.

Reply:
what do you mean ? you prefer i repeat the same test instead do a sub if .

Reply:
and repeat code ?

Reply:
```
{{ if and (.Values.mysql.db) (eq .Values.allDatabases.enabled false) (eq .Values.allDatabases.singleBackupFile false) }}
```

Reply:
what happen to `for MYSQL_DB in $MYSQL_DBS; do` ??

Reply:
i repeat it, one for mysql.db and one for allDatabases.enabled ?
why ?